config/locales/client.en.yml

```diff
@@ -53,6 +54,7 @@ en:
     motion_closed_by_expiry: "Proposal has closed"
     new_vote: "%{position} on the proposal"
     new_comment: "commented on the discussion"
+    comment_replied_to: "replied to your comment in the discussion"
```

Comment:
yep +1. as a rule, it's much easier to translate long strings with many
variables. wherever possible the translation string should be a whole
sentence.

On 2 February 2015 at 18:49, Robert Guthrie <notifications@github.com>
wrote:

> In config/locales/client.en.yml
> <https://github.com/loomio/loomio/pull/1863#discussion_r23907859>:
>
> > @@ -53,6 +54,7 @@ en:
> >      motion_closed_by_expiry: "Proposal has closed"
> >      new_vote: "%{position} on the proposal"
> >      new_comment: "commented on the discussion"
> > +    comment_replied_to: "replied to your comment in the discussion"
>
> Please include the whole message in the translation string.. user names
> and discussion titles
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/1863/files#r23907859>.
>



-- 

Richard D. Bartlett
Loomio co-founder
rich@loomio.org

<http://loomio.org/>